docs/source/classes_builders.rst

```diff
@@ -0,0 +1,41 @@
+Class Reference for Builders
+****************************
+
+Metadata Builders
+=================
+
+Note: Any Class with the Prefix **Mdsol** represents a Medidata Rave specific extension
```

Comment:
Since this is rst do you want to make these:

`..note:: Any Class with the Prefix **Mdsol** represents a Medidata Rave specific extension` ?

Reply:
Updated!